packages/node-utils/src/http.ts

```diff
@@ -65,7 +67,7 @@ type BaseEvents = {
 export class HttpServer<T extends EventMap> extends TypedEmitter<T & BaseEvents> {
     server: http.Server;
     private routes: Route[] = [];
-    private logger: Logger;
+    public logger: Logger;
```

Comment:
Nit: it is a good practice in TS classes to have public members first and then private instead of  public and private mixed.